snomed/com.b2international.snowowl.snomed.importer.rf2/src/com/b2international/snowowl/snomed/importer/rf2/util/ImportUtil.java

```diff
@@ -604,4 +605,8 @@ public static long parseLong(final String componentId) {
 
 		return ImmutableList.copyOf(listOfFiles);
 	}
+	
+	public static String createFeatureToggleString(String repositoryUUId, IBranchPath branchPath) {
+		return String.format("%s%s.import", repositoryUUId, branchPath.getPath());
```

Comment:
The repository ID and the path should also be separated by a character in the format string.